packages/swapper/src/swappers/zrx/getZrxTradeQuote/getZrxTradeQuote.test.ts

```diff
@@ -1,26 +1,38 @@
 import { ethereum } from '@shapeshiftoss/chain-adapters'
+import { AxiosStatic } from 'axios'
 import Web3 from 'web3'
 
 import { ZrxSwapper } from '../..'
 import { bn, bnOrZero } from '../../utils/bignumber'
 import { normalizeAmount } from '../../utils/helpers/helpers'
 import { setupQuote } from '../../utils/test-data/setupSwapQuote'
-import { zrxService } from '../utils/zrxService'
+import { baseUrlFromChainId } from '../utils/helpers/helpers'
+import { zrxServiceFactory } from '../utils/zrxService'
 
-const axios = jest.createMockFromModule('axios')
-//@ts-ignore
+const axios: AxiosStatic = jest.createMockFromModule('axios')
 axios.create = jest.fn(() => axios)
+
+jest.mock('../utils/zrxService', () => ({
+  zrxServiceFactory: () => axios.create()
+}))
+
+const zrxService = zrxServiceFactory('https://api.0x.org/')
+
 jest.mock('../utils/helpers/helpers')
 jest.mock('../../utils/helpers/helpers')
 jest.mock('../utils/zrxService')
 
 describe('getZrxTradeQuote', () => {
   const sellAmount = '1000000000000000000'
   ;(normalizeAmount as jest.Mock<unknown>).mockReturnValue(sellAmount)
+  ;(baseUrlFromChainId as jest.Mock<unknown>).mockReturnValue('eip155:1')
```

Comment:
should be a baseUrl to be more accurate

Reply:
Good catch. Addressed in https://github.com/shapeshift/lib/pull/850/commits/d4600424146891af210c68410417de33703067b2.